<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="statek_name" default="$(eval open(find('statek_config') + '/name').read())"/>

    <!-- Launch differential to twist feedback converter -->
    <group ns="$(arg statek_name)/converters">
        <node name="differential_to_twist" pkg="statek_hw" type="differential_to_twist_feedback_node.py">
            <param name="statek_name" type="string" value="$(arg statek_name)" />
            <rosparam command="load" file="$(find statek_config)/yaml/motion_config.yaml"/>
        </node>
    </group>

</launch>
